platform/source/lib/ax/platform/container_specs.py

```diff
@@ -103,7 +104,7 @@ def __init__(self, customer_image, namespace, version):
         # Sidecar needs to manage logs so add the log path here
         logpath = ContainerVolume("containerlogs", "/logs")
         if Cloud().in_cloud_aws():
-            logpath.set_type("HOSTPATH", "/mnt/ephemeral/docker/containers")
+            logpath.set_type("HOSTPATH", "/var/lib/docker/containers")
```

Comment:
does this work with argo-provisioned cluster as well?

Reply:
Yes. On Argo provisioned clusters, ```/var/lib/docker``` is a symlink to ```/mnt/ephemeral/docker```